R/textmodel_lr.R

```diff
@@ -0,0 +1,215 @@
+#' Logistic regression classifier for texts
+#'
+#' Fit a fast logistic regression classifier for texts, using the \pkg{glmnet}
+#' package.
+#' @param x the \link{dfm} on which the model will be fit.  Does not need to
+#'   contain only the training documents.
+#' @param y vector of training labels associated with each document identified
+#'   in \code{train}.  (These will be converted to factors if not already
+#'   factors.)
+#' @param nfolds .
```

Comment:
I'm trying to come up with a consistent framework for this that works for all textmodels. See https://github.com/quanteda/quanteda.classifiers/pull/24 where we are experimenting with `crossval()` that would work on a previously fitted model. Interested in your thoughts on this. But my preference is to keep it consistent rather than let each sub-function do it their own way.

Reply:
I assume you mean the `nfolds` argument? The cross-validation here is different from the one you have in mind. The goal is to find the best lambda value for the regression. If this is not done the user has to provide one her/himself, which is not ideal. I'm not entirely sure I got all the details right but point is it doesn't work out of the box if `glmnet()` is called instead of `cv.glmnet`.